modules/tinymce/src/themes/silver/main/ts/ui/statusbar/ResizeHandle.ts

```diff
@@ -11,14 +11,14 @@ import { SugarPosition } from '@ephox/sugar';
 
 import Editor from 'tinymce/core/api/Editor';
 
+import * as Options from '../../api/Options';
 import { UiFactoryBackstageProviders } from '../../backstage/Backstage';
 import * as Icons from '../icons/Icons';
 import { resize, ResizeTypes } from '../sizing/Resize';
 
 const getResizeType = (editor: Editor): ResizeTypes => {
   // If autoresize is enabled, disable resize
-  const fallback = !editor.hasPlugin('autoresize');
```

Comment:
What effects does this have on resizing and the autoresize plugin?

Reply:
Oops the above comment was meant to be removed as well... it has no difference on functionality as this just moved to the default in when we register the option.